doc/source/releases/22.05_announce.rst

```diff
@@ -1,12 +1,618 @@
+    that handled that particular request will tell all others to also reload,
+    lock jobs, etc.
+    For connection examples, see http://docs.celeryproject.org/projects/kombu/en/latest/userguide/connections.html
+    
+    Without specifying anything here, galaxy will first attempt to use your
+    specified database_connection above.  If that's not specified either, Galaxy
+    will automatically create and use a separate sqlite database located in your
+    <galaxy>/database folder (indicated in the commented out line below).
+
+   to
+
+   ::
+
+    Galaxy uses AMQP internally for communicating between processes.  For
+    example, when reloading the toolbox or locking job execution, the process
+    that handled that particular request will tell all others to also reload,
+    lock jobs, etc.
+    For connection examples, see https://docs.celeryq.dev/projects/kombu/en/stable/userguide/connections.html
+    
+    Without specifying anything here, galaxy will first attempt to use your
+    specified database_connection above.  If that's not specified either, Galaxy
+    will automatically create and use a separate sqlite database located in your
+    <galaxy>/database folder (indicated in the commented out line below).
+
+
+-  mapping.galaxy.mapping.debug.desc has changed from
+
+   ::
+
+    Debug enables access to various config options useful for development
+    and debugging: use_lint, use_profile, and use_printdebug.  It also
+    causes the files used by PBS/SGE (submission script, output, and error)
+    to remain on disk after the job is complete.
+    
+    In addition, this will set uWSGI's `honour-stdin` option to `true`;
+    thus, preventing uWSGI from remapping stdin to `/dev/null` and
+    enabling debugging with tools like pdb. To keep uWSGI's default
+    setting, set `honor-stdin` to `false` in the `uwsgi` section of this
+    configuration file.
+
+   to
+
+   ::
+
+    Debug enables access to various config options useful for development
+    and debugging: use_lint, use_profile, and use_printdebug.  It also
+    causes the files used by PBS/SGE (submission script, output, and error)
+    to remain on disk after the job is complete.
+
+
+-  mapping.galaxy.mapping.heartbeat_interval.desc has changed from
+
+   ::
+
+    Control the period (in seconds) between dumps. Use -1 to disable. Regardless
+    of this setting, if use_heartbeat is enabled, you can send a Galaxy process
+    (unless running with uWSGI) SIGUSR1 (`kill -USR1`) to force a dump.
+
+   to
+
+   ::
+
+    Control the period (in seconds) between dumps. Use -1 to disable. Regardless
+    of this setting, if use_heartbeat is enabled, you can send a Galaxy process
+    SIGUSR1 (`kill -USR1`) to force a dump.
+
+
+-  mapping.galaxy.mapping.helpsite_url.required has changed from
+
+   ::
+
+    False
+
+   to
+
+   ::
+
+    True
+
+
+-  mapping.galaxy.mapping.log_level.desc has changed from
```

Comment:
I know this is automatically generated but perhaps we should exclude changes that are only whitespace?

Reply:
We can totally curate this, I just pushed a commit that dropped insignificant changes